packages/utilities/src/utilities/scrollIntoView/scrollIntoView.js

```diff
@@ -25,40 +33,32 @@ const viewportMargin =
   ).toString() +
   'px 0px';
 
+const options = {
+  rootMargin: viewportMargin,
+  threshold: 0,
+};
+
 const scrollIntoView = (selector, delay = '400ms', iterations = true) => {
   window.addEventListener(
     'load',
     () => {
       const elements = document.querySelectorAll(selector);
-
-      console.log(elements);
-
-      const options = {
-        rootMargin: viewportMargin,
-        threshold: 0,
-      };
+      const root = document.querySelector(':root');
 
       const observer = new IntersectionObserver(function handleIntersect(
         entries
       ) {
+        root.style.setProperty('--delay', delay);
```

Comment:
Probably we can simply let application set the CSS custom property in the application CSS?

Reply:
The func specs detailed an option to allow adopters to provide their custom delay, this is in case the adopter wants to use different delays for different components (e.g. target `boxA` for `3s` and `boxB` for `1s` or so). This way the duration can differ between components over setting a "global" delay.

Reply:
I see and the custom delay can be done by:

```
.app--some-area-containing-scroll-into-view {
  --dds-delay: 3s
}
```

Reply:
Ah I see! That's definitely a better way to allow the adopter to have their own delay within their own CSS file.